.gitignore

```diff
@@ -5,6 +5,8 @@
 !/var/cache
 /var/cache/*
 !var/cache/.gitkeep
+!/var/data
+/var/data/*
```

Comment:
I agree, this only makes sense if we also commit a `.gitkeep` file inside `var/data/`.